### PR TITLE
Doc 4119 - 4.9.3, 4.8.9, 4.7.9, and 4.6.11 Updates 

### PR DESCRIPTION
**Pull request by paulmoloneyr3**
Updates to release notes, checksums and legal info for 4.9.3, 4.8.9, 4.7.9, and 4.6.11. Doc change related to the node registration tool for 4.9.3. Also some fixes of existing issues.

### CODE REVIEW

**Review by yarivus — DISMISSED** [review]
LGTM

**Comment by nadinequinr3** [comment]
Do we definitely not need to differentiate enterprise/community in the titles?

**Comment by paulmoloneyr3** [comment]
>

File: content/en/platform/corda/4.9/enterprise/release-notes-enterprise.md [hunk 1]
```diff
@@ -17,6 +17,22 @@ weight: 1
...
+
+### Upgrade recommendation
+
+As a developer or node operator, you should upgrade to the [latest released version of Corda](../../../../../en/platform/corda/4.9/enterprise.html) as soon as possible. The latest Corda Enterprise release notes are on this page, and you can find the latest upgrade guide [here](../../../../../en/platform/corda/4.9/enterprise/upgrading-index.md).
+
+### Fixed issues
+
+The following issues were resolved in this patch release:
+
+* Previously, both Corda Nodes certificates possessed Certificate Authority power because the CA attribute was set to true. The node registration tool now has a new  option, ```--node-is-ca```, allowing a node legal identity certificate to be created where it is not a CA.
```
Comment (adelel1) [inline content/en/platform/corda/4.9/enterprise/release-notes-enterprise.md:32]: The option is wrong. I have updated the issue https://r3-cev.atlassian.net/browse/ENT-6859
With the correct details. Should be ....
-C, --node-identity-cert-not-ca

**Review by adelel1 — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/4.9/enterprise/release-notes-enterprise.md (hunk 1 above)
Comment (paulmoloneyr3) [reply content/en/platform/corda/4.9/enterprise/release-notes-enterprise.md]: Thanks Adel, have just pushed an update there.

**Review by nadinequinr3 — APPROVED** [review]

**Review by yarivus — COMMENTED** [review]
Thanks @paulmoloneyr3

**Pull request merged**